test/e2e_node/dra_test.go

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package e2enode
+
+import (
+	"context"
+	"os"
+	"path"
+	"path/filepath"
+	"time"
+
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"k8s.io/klog/v2"
+
+	"k8s.io/kubernetes/test/e2e/framework"
+	e2enode "k8s.io/kubernetes/test/e2e/framework/node"
+
+	"k8s.io/dynamic-resource-allocation/kubeletplugin"
+	testdriver "k8s.io/kubernetes/test/e2e/dra/test-driver/app"
+)
+
+const (
+	driverName             = "test-driver.cdi.k8s.io"
+	cdiDir                 = "/var/run/cdi"
+	endpoint               = "/var/lib/kubelet/plugins/test-driver/dra.sock"
+	pluginRegistrationPath = "/var/lib/kubelet/plugins_registry"
+	draAddress             = "/var/lib/kubelet/plugins/test-driver/dra.sock"
+)
+
+var _ = ginkgo.Describe("[sig-node] DRA Node [Feature:DynamicResourceAllocation]", func() {
```

Comment:
I was going to create it after this PR is merged, so that would run something that already exists. I don't know how to do it another way to be honest.

Reply:
It's possible to create the jobs first. The periodic job then simply doesn't run anything until we merge this PR while the pre-merge job can be used to test this PR.

Reply:
Started from presubmit job: https://github.com/kubernetes/test-infra/pull/29636
Will add periodic job after this one works.

Reply:
Added comment with an explanation how to run it locally. PTAL